app/src/main/res/values/attrs.xml

```diff
@@ -2,6 +2,9 @@
 <resources>
 
     <declare-styleable name="ValidateableEditTextPreference">
+        <attr name="hint" format="string" />
```

Comment:
```suggestion
        <attr name="android:hint"/>
```

That way you'll be able to use `android:hint` in `prefs.xml`.

Reply:
:thinking: I am unsure if using the `android:` namespace here is a good idea. It hides that these style attributes are not working as expected. Only because of the "fix" in the custom class makes them work.

Reply:
Why should the user care that `android:hint` is not working with `EditTextPreference`? Isn't fixing the limitations of `EditTextPreference` the only reason `StyleableEditTextPreference` exists? Where's the surprise?

Reply:
:heavy_check_mark: Changed in 8a1cdab07ba57e017e94c5ead9cb6d2309773bc2.